homeassistant/components/netatmo/sensor.py

```diff
@@ -156,7 +156,7 @@ class NetatmoSensorEntityDescription(SensorEntityDescription):
         netatmo_name="sum_rain_24",
         native_unit_of_measurement=UnitOfPrecipitationDepth.MILLIMETERS,
         device_class=SensorDeviceClass.PRECIPITATION,
-        state_class=SensorStateClass.TOTAL_INCREASING,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
I don't think this is correct. Rain will not "unfall", thus `total` makes no sense here.

If this is a moving window, the state class should be removed.

../Frenck

Reply:
I changed the rain sensors to MEASUREMENT, like all others, or do you want me to remove that completely for rain?

Reply:
See https://github.com/home-assistant/core/issues/111510, I see the value going down, maybe some water evaporates?

Reply:
Well that is the question. Why is the value going down in this case. We need to know why that is, in order to make the right decision in this case.

It seems like this PR is fighting/aiming to resolve an error, but an explanation for the error being there is the first place, is missing?

Reply:
Ok, I will make this draft and maybe  @cgtobi knows where to find the API docu from Netatmo explaining this